### PR TITLE
feat: subscribe to both mess and de-register from notifications.

### PR DESCRIPTION
**Pull request by rhnvrm**
Could not test the bot. Feedback would be great. 👍

### CODE REVIEW

File: messmenu.py [hunk 1]
```diff
@@ -52,31 +49,24 @@ def sendMenu(user_id, mess_choice):
...
-			message = message+dish.text+"\n" 
-		message = message + "*\nLunch*\n----------------\n"
-		for dish in details[2].find_all('p'):
-			#Get each dish
-			message = message+dish.text+"\n" 
-		message = message + "*\nDinner*\n----------------\n"
-		for dish in details[3].find_all('p'):
-			#Get each dish
-			message = message+dish.text+"\n"  
+	message = generateMenu(mess_choice, menuTable)
```
Comment (FlameFractal) [inline messmenu.py]: `generateMenu` generates based on time. So it'll only send breakfast if time<10am

`sendFullMenu` will no longer work, right? It was to get the whole menu of the day

File: messmenu.py [hunk 2]
```diff
@@ -23,18 +23,15 @@
...
 
-
-def sendMenu(user_id, mess_choice):
-	menuTable = (bs.BeautifulSoup(requests.get(messUrl).text,'lxml')).find_all(id='dh2MenuItems')
-	details = menuTable[mess_choice].find_all('td')
-	
-	message = "*Menu for DH 1*\n\n" if mess_choice == 0 else "*Menu for DH 2*\n\n"
+def generateMenu(mess, menuTable):
+	details = menuTable[mess].find_all('td')
+	message = "*Menu for DH +"+str(mess)+"*\n\n"
```
Comment (FlameFractal) [inline messmenu.py:28]: str(mess+1)

**Comment by FlameFractal** [comment]
Hey. Just keep the `sendFullMenu`'s code and change unregister to deregister. Else it's 👍

**Comment by rhnvrm** [comment]
@FlameFractal fixed and pushed

**Comment by FlameFractal** [comment]
Great!

**Pull request merged**